### PR TITLE
Adding multizonal test suites in sig-storage dashboard

### PR DESCRIPTION
**Pull request by verult**

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -3648,6 +3648,18 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-alpha-features
     base_options: 'include-filter-by-regex=Volume%7Cstorage'
     description: 'storage gci gce alpha features e2e tests for master branch'
+  - name: gci-gke-multizone
+    test_group_name: ci-kubernetes-e2e-gci-gke-multizone
+    base_options: 'include-filter-by-regex=Volume%7Cstorage'
+    description: 'storage gci gke multizone e2e tests for master branch'
+  - name: gke-multizone
+    test_group_name: ci-kubernetes-e2e-gke-multizone
```
Comment (msau42) [inline testgrid/config/config.yaml]: Is this different from the gci version?  Isn't everything on gci now?

Comment (krzyzacy) [reply testgrid/config/config.yaml]: yep.. they are running the same thing now, seems the multizone job is slipped from the cleanup, I'll delete one of them.

Comment (BenTheElder) [reply testgrid/config/config.yaml]: cc @krzyzacy ^
I think so?

**Comment by verult** [comment]
Deleted gke-multizone from sig-storage dashboard.

**Pull request merged**